### DISCUSSION TITLE
How do I configure two Google Drive accounts?

### DISCUSSION THREAD

**Discussion by rahul101001000** [dabbu-knowledge-platform/cli#4] [Q&A] [answered]
I have configured Dabbu successfully for one GDrive account, I would like to add another GDrive account as well. How should I do that? I can't seem to go back to the config screen. 

ps great concept, exactly what I was looking for.

**Comment by gamemaker1** [comment, marked as answer]
Hi there!

Thanks for using Dabbu, we're glad you're liking it!

Regarding you question, you can enter the following to create a new drive:

```sh
::
```

This will start the setup drive sequence. Enter a different drive name this time, but **enter the same client ID and secret as last time. The only different thing you should do is sign in with a different account in the browser.**

Hope this helps.

Thanks and regards,
gamemaker1

**Reply by gamemaker1** [reply]
Old answer, not valid as of v2.0.0

~~In case someone is looking at this discussion for creating a new drive for a different provider, here is a generic command for that:~~

~~`::<provider-id>`~~

~~Where the provider ID (as of now) can be either `hard_drive` or `google_drive`.~~
